package.json

```diff
@@ -118,6 +118,7 @@
     "serialize-error": "^1.1.0",
     "set-immediate-shim": "^1.0.1",
     "source-map-support": "^0.4.0",
+    "stack-utils": "github:tapjs/stack-utils#long-stack-traces",
```

Comment:
npm apparently adds this now, but it's not needed as github is the default for shorthands.